x-pack/plugins/security/server/routes/session_management/info.ts

```diff
@@ -32,8 +32,8 @@ export function defineSessionInfoRoutes({ router, logger, getSession }: RouteDef
 
         return response.noContent();
       } catch (err) {
-        logger.error(`Error retrieving user session: ${err.message}`);
```

Comment:
__nit:__ I think it doesn't make much sense to keep this `try/catch` and custom error message anymore, would you mind removing that all together? I believe in logs we'll be able to correlate the actual error with the request to that particular route?

Reply:
> I believe in logs we'll be able to correlate the actual error with the request to that particular route?

yes, in the logged error stack